app/code/Magento/Sales/Model/Order/Shipment.php

```diff
@@ -93,6 +101,16 @@ class Shipment extends AbstractModel implements EntityInterface, ShipmentInterfa
      */
     protected $orderRepository;
 
+    /**
+     * @var ShipmentCommentRepositoryInterface
+     */
+    protected $shipmentRepository;
+
+    /**
+     * @var SearchCriteriaBuilder
+     */
+    protected $searchBuilder;
```

Comment:
Just a side note: I would suggest to use `searchCriteriaBuilder` as variable name.
You are not building a search, but a search criteria.

Reply:
I agree with it but some other developer pushed commit with changes to names of variables. Can I remove his commit?
I am new to contributing so I am not sure what the process exactly.